### DISCUSSION TITLE
How to add alternative text for screenreader on a TextBlock?

### DISCUSSION THREAD

**Discussion by hallivia** [AvaloniaUI/Avalonia#14252] [General]
<TextBlock Focusable="True" Text="{Binding RegisterLabel}" HorizontalAlignment="Center" />

Hi there, how do I add alternative text for the screenreader to read? I tried adding AutomationProperties.HelpText and other AutomationPropeties properties and that didn't help. What is the best practice if we want to add a different text for narrator to read that is not the same as the value of the Text?

**Comment by maxkatz6** [comment]
Isn't it AutomationProperties.Name ? But I am afraid it might be ignored in case of TextBlock/Label controls, as they override this property.

**Comment by hallivia** [comment]
Hi maxkatz6, I tried that as well and it doesn't work. I'll open a feature request for this then.
